docs/api.md

```diff
@@ -545,7 +545,7 @@ kinto.events.on("deprecated", function(event) {
 
 Transformers are basically hooks for encoding and decoding records, which can work synchronously or asynchronously.
 
-For now, only *remote transformers* have been implemented, but there are plans to implement local transformers and collection transformers in a next version.
+For now, only *remote transformers* and *collection transformers* have been implemented, but there are plans to implement local transformers in a next version.
```

Comment:
Also, I think it could be worth mentioning this feature in the «extending kinto.js» docs, no?